packages/material-ui/src/styles/experimentalStyled.js

```diff
@@ -106,10 +107,12 @@ const experimentalStyled = (tag, options, muiOptions = {}) => {
       return '';
     });
 
-    expressionsWithDefaultTheme.push((props) => {
-      const theme = isEmpty(props.theme) ? defaultTheme : props.theme;
-      return styleFunctionSx({ ...props, theme });
-    });
+    if (!skipSx) {
+      expressionsWithDefaultTheme.push((props) => {
+        const theme = isEmpty(props.theme) ? defaultTheme : props.theme;
+        return styleFunctionSx({ ...props, theme });
```

Comment:
I might understand why the curried API on styled-system, maybe it's so they don't need to spread the props, I wonder if it helps with performance:

```suggestion
        return styleFunctionSx(props)(theme);
```

Reply:
Let me test this separately, wouldn't like to break something else somewhere :)))

Reply:
But looks like it really may help